source/manual/business-readiness-publish-changes.html.md

```diff
@@ -5,7 +5,7 @@ section: Business readiness finder
 layout: manual_layout
 parent: "/manual.html"
 important: true
-last_reviewed_on: 2019-05-01
```

Comment:
I'd fix up this change, and the one below to the commit where the content has changed.